contracts/erc20-gateway/ERC20Cogateway.sol

```diff
@@ -30,6 +32,27 @@ contract ERC20Cogateway is
     MessageBus,
     ERC20GatewayBase {
 
+    /** Events */
+
+    /** Emitted when withdraw message is declared */
+    event WithdrawIntentDeclared(
```

Comment:
Can we update this as below:
```
    event WithdrawIntentDeclared(
        uint256 amount,
        uint256 nonce,
        address beneficiary,
        uint256 feeGasPrice,
        uint256 feeGasLimit,
        address withdrawer,
        address utilityToken,
        bytes32 messageHash
    );
```

I will also add a comment in the ticket.

Reply:
Updated.